src/components/SchemaRow/TopLevelSchemaRow.tsx

```diff
@@ -17,12 +19,18 @@ export const TopLevelSchemaRow = ({ schemaNode }: Pick<SchemaRowProps, 'schemaNo
   const childNodes = React.useMemo(() => calculateChildrenToShow(selectedChoice.type), [selectedChoice.type]);
   const nestingLevel = 0;
 
+  const internalSchemaError = getInternalSchemaError(schemaNode);
+
   // regular objects are flattened at the top level
   if (isRegularNode(schemaNode) && isPureObjectNode(schemaNode)) {
     return (
       <>
         <ScrollCheck />
         <ChildStack schemaNode={schemaNode} childNodes={childNodes} currentNestingLevel={nestingLevel} />
+        {internalSchemaError.hasError && (
+          // TODO (JJ): Add mosaic tooltip showing ref error
```

Comment:
Do we still need this todo?

Reply:
Ooop - good catch - I removed the todo from both places!